pkg/extensions/scrub/scrub.go

```diff
@@ -58,6 +59,6 @@ func NewTask(imgStore storageTypes.ImageStore, repo string, log log.Logger) *Tas
 	return &Task{imgStore, repo, log}
 }
 
-func (scrubT *Task) DoWork() error {
-	return RunScrubRepo(scrubT.imgStore, scrubT.repo, scrubT.log)
+func (scrubT *Task) DoWork(ctx context.Context) error {
+	return RunScrubRepo(scrubT.imgStore, scrubT.repo, scrubT.log) //nolint: contextcheck
```

Comment:
why not pass it to the scrub task as well?

Reply:
done